mipengine/node/tasks/merge_tables.py

```diff
@@ -19,33 +20,31 @@ def get_merge_tables(context_id: str) -> str:
 
         Returns
         ------
-        str(List[str])
-            A list of merged table names in a jsonified format
+        List[str]
+            A list of merge table names
     """
-    return json.dumps(merge_tables.get_merge_tables_names(context_id))
+    return merge_tables.get_merge_tables_names(context_id)
 
```

Comment:
it does not return a list of strings. It returns a string. This string looks like
`" 'mergetable1','mergetable2','mergetable3'... "` 
but it will not be a list of strings in the sense of the python object List

Reply:
No, it returns a List of strings in the sense of python object list.

Celery can automatically encode/decode primitive types without the need of json serializing.

We have verified this. Can you check this on the controller as well?